doc/api/globals.md

```diff
@@ -592,6 +592,7 @@ added: v22.4.0
 
 A browser-compatible implementation of [`localStorage`][]. Data is stored
```

Comment:
I think this needs to mention that both `localStorage` and `sessionStorage` are not per user or per request (when in the context of a server).